stable/shell/templates/shell-configmap.yaml

```diff
@@ -46,6 +46,20 @@ data:
     fi
 {{- end }}
 
+    # Create /tmp in default container if not exists and set premissions to 777
+    set +e
```

Comment:
refrain from disabling the error check. You don't need to check the status like this.
e.g. `hadoop fs -test -e /tmp || hadoop fs -mkdir /tmp`